setup/setuputils.class.inc.php

```diff
@@ -1878,8 +1878,8 @@ static public function GetLicenses($sEnv = null)
 		$aLicenceFiles = glob(APPROOT.'setup/licenses/*.xml');
 		if (empty($sEnv))
 		{
-			$aLicenceFiles = array_merge($aLicenceFiles, glob(APPROOT.'datamodels/*/*/license.*.xml'));
-			$aLicenceFiles = array_merge($aLicenceFiles, glob(APPROOT.'extensions/*/license.*.xml'));
+			$aLicenceFiles = array_merge($aLicenceFiles, glob(APPROOT.'datamodels/*/{*,*/*}/license.*.xml', GLOB_BRACE));
```

Comment:
are there also multi modules in the iTop core data models?

Reply:
Not yet, and this shouldn't happen
I'm not sure this is worse it to add the pattern here as well ?

Reply:
Well, its a bit strange to support it while it should not happen. If you support it, you imply it might happen.

Reply:
The datamodels/2.x folder has always contained modules.
Extensions are a package of one or many modules. This package is intended for easier delivery by the iTop Hub or ITSM Designer, or to have just one download for manual installation. 
Installing manually extensions in datamodels/2.x is not something we want to support.

Reply:
So the pattern should be stayed as
```suggestion
			$aLicenceFiles = array_merge($aLicenceFiles, glob(APPROOT.'datamodels/*/*/license.*.xml'));
```

Reply:
Sorry I did too much tasks and didn't get right your comments :/
I made a mistake in the initial commit, I fixed it with 59465b50

Reply:
closing this conversation as well after Thomas reaction + approval